### PR TITLE
CSC Methods for Solving Systems of Linear Equations

### PR DESCRIPTION
**Pull request by TheBaxes**
Needs testing

### CODE REVIEW

File: CSC.cpp [hunk 1]
```diff
@@ -53,4 +54,103 @@ vector<double> CSC::mul(vector<double> x) {
...
+    return x;
+  }
+  throw "rip";
+}
+
 CSC::CSC(int n_rows_, int n_cols_) : SparseMatrix::SparseMatrix(n_rows_, n_cols_) {}
+
+//          i c f g a e d h b j
+//IA rows = 6 2 4 5 1 3 2 5 1 6/
+//JA cols = 1 2 4 5 7 9 11
```
Comment (Essux) [inline CSC.cpp:156]: Should we leave this?

**Comment by Essux** [comment]
Did you test it with the System of Equations generator?

File: CSC.cpp (hunk 1 above)
Comment (TheBaxes) [reply CSC.cpp:156]: No. I forgot that I should remove that.

**Comment by TheBaxes** [comment]
> Did you test it with the System of Equations generator?

No, but I did a test case and compare it with MSR. It failed

**Pull request closed without merging**